scala/spark-wordcount/README.md

```diff
@@ -38,10 +52,39 @@ Create a Cloud DataProc instance:
 
 ## Build the jar
 
-The Spark job is assembled into a fat jar with all of its dependencies. To build, run:
+The Spark application is assembled into a fat jar with all of its dependencies. To build, run:
 
     mvn assembly:assembly
```

Comment:
maybe add `clean` to ensure clean builds

Reply:
fixed